app/services/markdown_processor/fixer/base.rb

```diff
@@ -0,0 +1,95 @@
+module MarkdownProcessor
+  module Fixer
+    # Services here in the Fixer module should inherit from this base class.
+    # #call is implemented here so other services only need to define a
+    # METHODS constant that is an Array of symbols referencing other methods
+    # found here.
+    #
+    # For example
+    # METHODS = %i[add_quotes_to_tile add_quotes_to_description]
+    class Base
+      FRONT_MATTER_DETECTOR = /-{3}.*?-{3}/m.freeze
+
+      def self.call(markdown)
+        return unless markdown
+
+        self::METHODS.reduce(markdown) { |acc, elem| public_send(elem, acc) }
+      end
+
+      def self.add_quotes_to_title(markdown)
+        add_quotes_to_section(markdown, section: "title")
+      end
+
+      def self.add_quotes_to_description(markdown)
+        add_quotes_to_section(markdown, section: "description")
+      end
+
+      # This turns --- into ------- after the first two,
+      # because --- messes with front matter
+      def self.modify_hr_tags(markdown)
+        markdown.gsub(/^---/).with_index { |match, i| i > 1 ? "#{match}-----" : match }
+      end
+
```

Comment:
We didn't have any specs for this method and when I went to write them...I realized this method may have actually not been working this entire time 🙈 . Since we don't currently have any specs for it, my understanding of what this is supposed to do is based on the comment here. I updated this logic, please let me know if I misunderstood the purpose of this method!